api/healthcheck.go

```diff
@@ -10,32 +10,53 @@
 package api
 
 import (
+	"context"
 	"fmt"
 	"net/http"
 	"strings"
 
-	"github.com/labstack/echo"
+	"go.uber.org/zap"
+
+	api "github.com/topfreegames/podium/proto/podium/api/v1"
 )
 
-// HealthCheckHandler is the handler responsible for validating that the app is still up
-func HealthCheckHandler(app *App) func(c echo.Context) error {
-	return func(c echo.Context) error {
-		c.Set("route", "Healthcheck")
-		workingString := app.Config.GetString("healthcheck.workingText")
+//healthCheckHandler is the handler responsible for validating that the app is still up
```

Comment:
nit: Add a space before the start of the comment, full stop at the end.

`// healthCheckHandler (...) is still up.`